com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/ApplicationURI.java

```diff
@@ -81,6 +81,8 @@ public void setURIPart(String partName, String part) {
             case SupportedParameters.ROUTE_PATH:
                 setPath(part);
                 break;
+            default:
+                // In default case just exit the method without setting anything
```

Comment:
Maybe a return statement instead of a comment? This way you'll be sure you've exited because if you add something after the switch this comment will be misleading.

Reply:
done